agent/acs/handler/acs_handler.go

```diff
@@ -97,6 +93,7 @@ type session struct {
 	ctx                             context.Context
 	cancel                          context.CancelFunc
 	backoff                         retry.Backoff
+	metricsFactory                  metrics.EntryFactory
```

Comment:
nit: We can replace usages of the no-op metricsFactory defined on L260 with this new one so that we only have a single no-op metricsFactory being used.

Reply:
Sure.